process/interceptors/singleDataInterceptor.go

```diff
@@ -10,53 +10,62 @@ import (
 	"github.com/ElrondNetwork/elrond-go/process"
 )
 
+// ArgSingleDataInterceptor is the argument for the single-data interceptor
+type ArgSingleDataInterceptor struct {
+	Topic            string
+	DataFactory      process.InterceptedDataFactory
+	Processor        process.InterceptorProcessor
+	Throttler        process.InterceptorThrottler
+	AntifloodHandler process.P2PAntifloodHandler
+	WhiteListRequest process.WhiteListHandler
+	CurrentPeerId    core.PeerID
+}
+
 // SingleDataInterceptor is used for intercepting packed multi data
 type SingleDataInterceptor struct {
 	topic                      string
 	factory                    process.InterceptedDataFactory
 	processor                  process.InterceptorProcessor
 	throttler                  process.InterceptorThrottler
-	whiteListRequested         process.WhiteListHandler
+	whiteListRequest           process.WhiteListHandler
```

Comment:
whiteListHandler ?

Reply:
done, remained from the real resolvers where we had 2 white list handlers